src/main/webapp/assets/modules/email-setting/emailSetting.js

```diff
@@ -0,0 +1,50 @@
+techlooper.controller("emailSetting", function ($scope, utils, jsonValue, apiService, validatorService) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'validatorService' is defined but never used.